test/AST-Quake/vector-ctor-initlist-int.cpp

```diff
@@ -0,0 +1,56 @@
+/*******************************************************************************
```

Comment:
Super nitpicky: I think all the files in the AST-Quake directory use `_` instead of `-` ... so for consistency ...